lib/router/layer.js

```diff
@@ -15,13 +15,7 @@
 
 var pathRegexp = require('path-to-regexp');
 var debug = require('debug')('express:router:layer');
-
-/**
- * Module variables.
- * @private
- */
-
-var hasOwnProperty = Object.prototype.hasOwnProperty;
+var hasOwnProperty = require('../utils').hasOwnProperty;
```

Comment:
This change isn't strictly related, but I figured we could use the new utility. Happy to revert.